vendor/github.com/openshift/api/operator/v1/0000_70_cluster-network-operator_01.crd.yaml-patch

```diff
@@ -1,3 +0,0 @@
-- op: add
-  path: /spec/versions/name=v1/schema/openAPIV3Schema/properties/spec/properties/defaultNetwork/properties/ovnKubernetesConfig/properties/policyAuditConfig/default
-  value: {}
```

Comment:
why did this go away?

Reply:
https://github.com/openshift/api/pull/1102